conversions/bin_to_octal

```diff
@@ -0,0 +1,43 @@
+"""
+The function below will convert any binary string to the octal equivalent.
+
+>>> bin_to_octal("1111")
+'17'
+
```

Comment:
```suggestion
```

Reply:
There has to be some doctests in the code